dataplane.code-workspace

```diff
@@ -164,6 +164,10 @@
       "name": "perfstress",
       "path": "sdk/test-utils/perfstress"
     },
+    {
+      "name": "batch",
+      "path": "sdk/batch/batch"
+    },    
```

Comment:
@ramya-rao-a @xirzec @bterlson technically, batch is a dataplane package but I guess this workspace is meant for track2 packages, no?

Reply:
Well, this is mainly for the handwritten packages + handwritten over auto generated packages that share the same build and release pipeline system to make it easier to work across packages with similar needs. So, yes, the change to this file can be reverted as well

Reply:
I will undo it.